apps/omg_watcher_info/test/support/factory.ex

```diff
@@ -105,19 +105,19 @@ defmodule OMG.WatcherInfo.Factory do
     - To override `creating_transaction`, also consider overriding `txindex` and `oindex`.
     - To override `spending_transaction`, also consider overriding `spending_tx_oindex`
   """
-  def txoutput_factory() do
+  def txoutput_factory(attrs) do
     block = build(:block)
     transaction = build(:transaction, block: block)
 
     %DB.TxOutput{
       blknum: block.blknum,
       txindex: 0,
-      oindex: 0,
+      oindex: Map.get(attrs, :oindex, 0),
```

Comment:
Actually ExMachina will auto override the values if the keys match so you don't even need Map.get stuff here.

Kind of obscure at first, but it's one of the big uses of ExMachina.